frontend/tests/playwright/fixtures/trace-override.ts

```diff
@@ -0,0 +1,29 @@
+import { TestInfo } from '@playwright/test';
+import path from 'node:path';
+
+export function traceOutputDirTemplate(testInfo: TestInfo) {
+   // The path to where all the trace files will be outputted as defined
+   // in the Playwright config
+   const globalOutputDir = testInfo.project.outputDir;
+
+   // titlePath houses the path to the test file relevant to the test directory
+   // e.g. 'product/add_to_cart.spec.ts' -> 'product/add_to_cart'
+   const testFile = testInfo.titlePath[0].replace(/\.spec\.ts$/, '');
```

Comment:
I know we only have `spec.ts` files right now. Do you think that will change? Maybe we add `spec.tsx` at some point?

Reply:
> I know we only have spec.ts files right now. Do you think that will change?

I don't think it will change, there might be a chance someone who adds a `.test.ts` file but that would assume they didn't notice the common usage of `.spec.ts` and we can comment on in CR.

As for the `spec.tsx`, the usage of `tsx` would imply that there are some `react` elements used in the test file which `playwright` is not capable of parsing. This is what caused this issue https://github.com/iFixit/react-commerce/issues/1400

Reply:
Sounds good :)